.circleci/config.yml

```diff
@@ -1,65 +1,142 @@
 version: 2
 jobs:
   lint:
-    machine: true
+    docker:
+      - image: quay.io/deis/lightweight-docker-go:v0.2.0
+    environment:
+      SKIP_DOCKER: true
+      CGO_ENABLED: 0
+    working_directory: /go/src/github.com/Azure/open-service-broker-azure
```

Comment:
You'll see these previous 6 lines recur throughout this file. We want to use a specific Docker image for a container executor, instruct any make targets used by the job that they're _already_ running in an acceptable container so the command won't be "wrapped" in `docker run`, and we disable cgo, cause ain't nobody like cgo, amiright?

Reply:
It would be nice if I could DRY this up, but there's no way to dry up YAML. C'est la vie.